devito/dse/inspection.py

```diff
@@ -125,16 +70,21 @@ def estimate_cost(handle, estimate_external_functions=False):
             pass
     try:
         # At this point it must be a list of SymPy objects
-        # We don't count non floating point operations
+        # We don't use SymPy's count_ops because we do not count integer arithmetic
+        # (e.g., array index functions such as i+1 in A[i+1])
+        # Also, the routine below is *much* faster than count_ops
```

Comment:
Possibly beyond the scope of this PR, but how well is the flops estimation tested atm?